packages/application/src/index.ts

```diff
@@ -506,6 +506,8 @@ export class Application<T extends Widget = Widget> {
     // Mark the application as started;
     this._started = true;
 
+    this._bubblingKeydown = options.bubblingKeydown || false;
```

Comment:
Could you add a public getter/setter so that it can be change dynamically for testing? Marking those as @experimental or even @ignore to keeping then secret.